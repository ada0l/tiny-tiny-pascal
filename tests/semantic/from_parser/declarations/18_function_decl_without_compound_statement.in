function a(): Integer;

begin end.
