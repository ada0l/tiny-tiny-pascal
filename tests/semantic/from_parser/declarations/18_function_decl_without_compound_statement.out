(3, 10): Expected  SEMICOLON; but found .